app/views/capacity.html

```diff
@@ -40,6 +40,71 @@
 
 <div id="plotly-div-line" role="presentation"></div>
 
+{% if organisationLevel !== 'offender-manager' %}
+  <h3 class="heading-medium">Capacity breakdown</h3>
+  <table id="example" class="data-table dataTable">
+    <thead>
+      <tr>
+        <th colspan="2" id="team">{{ childOrganisationLevelDisplayText }}</th>
+        <th class="" colspan="14" id="breakdown">Capacity breakdown</th>
+      </tr>
+      <tr>
+        <th id="name">Name</th>
+        <th id="grade">Grade</th>
+        <th id="capacity">Capacity(%)</th>
+        <th id="cases">Cases</th>
+        <th id="paroms"><abbr title="Parole assessment reports by the Offender Manager">PAROMs</abbr></th>
+        <th id="fdrs"><abbr title="Fast delivery reports">FDRs</abbr></th>
+        <th id="sdrs"><abbr title="Standard delivery reports">SDRs</abbr></th>
+        <th id="arms"><abbr title="Active Risk Management System">ARMS</abbr></th>
```

Comment:
All of the rest of these abbreviation expansions only capitalise the first word. I think capitalising each make sense but consistency is key.

Reply:
Changed to capitalise the abbreviation words